### PR TITLE
Add exercise Connect

### PR DESCRIPTION
**Pull request by CRivasGomez**
Hey!
I added Connect exercise 🙂

### CODE REVIEW

File: exercises/connect/connect.test.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import Board from './connect'
```
Comment (masters3d) [inline exercises/connect/connect.test.ts]: An you name this Connect or ConnectBoard ?

File: exercises/connect/connect.test.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        const board = [
+            'O X X X X X X X X',
+            ' O X O O O O O O O',
+            '  O X O X X X X X O',
+            '   O X O X O O O X O',
+            '    O X O X X X O X O',
+            '     O X O O O X O X O',
+            '      O X X X X X O X O',
+            '       O O O O O O O X O',
+            '        X X X X X X X X O',
```
Comment (masters3d) [inline exercises/connect/connect.test.ts]: Why are the boards skewed this was?

File: exercises/connect/connect.test.ts (hunk 1 above)
Comment (CRivasGomez) [reply exercises/connect/connect.test.ts]: `ConnectBoard` looks good for me.

File: exercises/connect/connect.test.ts (hunk 2 above)
Comment (CRivasGomez) [reply exercises/connect/connect.test.ts]: According to canonical data associated to this exercise is how the test input data is supposed to look like.

**Pull request merged**